tests/staged_sim_and_verif/staged_sim_and_verif.sby

```diff
@@ -0,0 +1,14 @@
+[options]
+mode cover
+depth 1
+
+[engines]
+smtbmc
+
+[script]
+# Minimal job so dumptaskinfo picks up the tools this flow requires.
```

Comment:
Is this a separate file so that `dumptaskinfo` only registers a single task to ensure sequential execution?  (as opposed to calling the script `skip_staged_flow.sh` and having it be called for each task)

Reply:
Ah, actually, the issue I'm running into now is that, even if the test harness can handle a multi-task SBY file, it treats each task as being independent of the other. So if my task1 is dependent on verific, and task2 is dependent on task1 (but not verific), then CI will attempt to run only task2 when verific is not available and fail because task1 has not been run. The hack gets around this by essentially grouping all four tasks into a single SBY task via the dummy SBY file.

Reply:
Without changing the test harness to allow overriding multiple tasks or specifying dependencies, I think this is probably the correct approach